opensearch-api/spec/opensearch/api/actions/create_pit_spec.rb

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+require 'spec_helper'
+
+describe 'client#create_pit' do
+  let(:expected_args) do
+    [
+      'POST',
+      'movies,books/_search/point_in_time',
```

Comment:
wait, this seems wrong!

Reply:
Yes it does: https://opensearch.org/docs/latest/search-plugins/point-in-time-api/#path-parameters